.changeset/fair-boxes-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'penpot-exporter': minor
+---
+
+Remove custom font form
```

Comment:
Maybe clarify the reasons to explain why we are removing this. Just to clarify that we are not removing any feature, just simplifying the process.